pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -148,37 +135,45 @@ func SetDefaults_KubeControllerManagerConfiguration(obj *KubeControllerManagerCo
 	}
 }
 
-func SetDefaults_GenericComponentConfiguration(obj *GenericComponentConfiguration) {
+func SetDefaults_CloudControllerManagerConfiguration(obj *CloudControllerManagerConfiguration) {
 	zero := metav1.Duration{}
-	if obj.MinResyncPeriod == zero {
-		obj.MinResyncPeriod = metav1.Duration{Duration: 12 * time.Hour}
+	if obj.ServiceController.ConcurrentServiceSyncs == 0 {
+		obj.ServiceController.ConcurrentServiceSyncs = 1
 	}
-	if obj.ContentType == "" {
-		obj.ContentType = "application/vnd.kubernetes.protobuf"
+	if obj.NodeStatusUpdateFrequency == zero {
+		obj.NodeStatusUpdateFrequency = metav1.Duration{Duration: 5 * time.Minute}
 	}
-	if obj.KubeAPIQPS == 0 {
-		obj.KubeAPIQPS = 20.0
+	// TODO: Is there a reason the ccm should have non-default/recommended qps/burst values or can we just remove this?
```

Comment:
yes, I'd keep qps/burst per component for now, with the existing values like this

Reply:
I'll remove the TODO comment, and instead clarify why we do this